mail_tracking/models/mail_message.py

```diff
@@ -87,7 +88,7 @@ def _get_email_cc(self):
                 for partner in partners:
                     email_cc_list.discard(partner.email)
                     email_cc_list_checked.append(
-                        (partner.email, [partner.id, partner.display_name]))
+                        (partner.email, [partner.id, partner.name]))
```

Comment:
Why? :thinking:

Reply:
Because "partner_tracking" only prints the name too: 
https://github.com/OCA/social/blob/be273331180f017b931231049a097159ca945a6f/mail_tracking/models/mail_message.py#L52
https://github.com/OCA/social/blob/be273331180f017b931231049a097159ca945a6f/mail_tracking/models/mail_message.py#L68